subsys/zigbee/osif/zb_nrf_platform.c

```diff
@@ -76,6 +79,13 @@ K_THREAD_STACK_DEFINE(zboss_stack_area, CONFIG_ZBOSS_DEFAULT_THREAD_STACK_SIZE);
 static struct k_thread zboss_thread_data;
 static k_tid_t zboss_tid;
 
+/**@brief Gets pointer to zboss thread struct.
+ */
+k_tid_t zb_get_zboss_thread_id(void)
```

Comment:
I feel that we should not expose zboss thread id unless we need to. This can easily get abused.